packages/react/src/fonts.scss

```diff
@@ -1 +1 @@
-@use "@fontsource-variable/roboto-flex";
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Flex:opsz,wght@8..144,100..1000&display=swap');
```

Comment:
I have a bit of a conscious problem hard coding google urls in there, WDYT @jbpenrath? Plus if you don't want Roboto do not import it? You can remove `@use "@openfun/cunningham-react/sass/fonts"` of your sass file. And use your custom font instead the way you intend it?

Reply:
We can schedule a quick meeting of few minutes to discuss this with @AntoLC @jbpenrath, wdyt guys?

Reply:
I'm also very uncomfortable with hardcoding google, from our perspective it's better to have a slight performance hit than to send tracking. This should be an option at least :)

Reply:
I am fine if we remove it from `ui-kit` so if you feel uncomfortable with google cdn, but it is not a slight performance hit, particularly on mobile:
<img width="885" height="498" alt="image" src="https://github.com/user-attachments/assets/006eb823-6326-4b24-a612-c2d685f46cbb" />
<img width="1271" height="515" alt="image" src="https://github.com/user-attachments/assets/ba633aa4-346d-443e-8ae3-d90aca7f27c9" />


The css is mainly base64 font, for a font that Docs do not use: https://docs.numerique.gouv.fr/_next/static/css/4abed4aecf4e527e.css

Reply:
Maybe what we want to do is just to simply remove this entry on the ui-kit package instead ? https://github.com/suitenumerique/ui-kit/blob/519c1e581f948887f368324a49b1e6e754a7d69c/src/library.scss#L1

Reply:
Yup we can quickly fix the performance issue encount by Anto by fixing ui-kit. But I'm convinced that we have also to drop this file from Cunnigham. This is a little helper for DX but in counter part this the worsest manner to embedded a font.

It represents a breaking change but as we are totally refactoring design tokens that represent also a breaking change I think we will have to include this change in the version 4 of Cunningham

Reply:
Cunningham already allows for custom font loading as the font file is standalone, along with the styles one. 

Instead the modification should be made up on the ui-kit, what we did there: https://github.com/suitenumerique/ui-kit/pull/102

Thank you Anthony for your contribution !